akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -32,8 +32,7 @@ object TwitterStreamQuickstartDocSpec {
   //#model
 
   //#tweet-source
-  val tweets: Source[Tweet, NotUsed]
-  //#tweet-source
+  val tweets: Source[Tweet, NotUsed] //#tweet-source
```

Comment:
ups, scalariform reformats this, breaks docs, remove this change from the commit